docs/apis-clients/console-api-reference.md

```diff
@@ -1,37 +1,39 @@
 ---
 id: console-api-reference
 title: Console API clients (REST)
-description: "Create and manage clusters, and interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 UI."
+description: "Create and manage clusters, and interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 Console."
 ---
 
 ## Console API (REST)
 
-For all requests, include the access token for Cloud API in the Authorization header: `authorization:Bearer ${TOKEN}`.
+For all requests, include the access token in the Authorization header: `authorization:Bearer ${TOKEN}`.
 
 :::note
 A detailed API description can be found [here](https://console.cloud.camunda.io/customer-api/openapi/docs/#/) via Swagger. With a valid access token, this offers an interactive API experience against your Camunda Platform 8 cluster.
 :::
 
-## Cloud API management
+### Client credentials and scopes
 
-To interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 UI, create Cloud API clients in the organization settings under the **Cloud Management API** tab.
+To interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 Console, create client credentials in the organization settings under the **Console API** tab.
 
-Cloud API clients are created for an organization, and therefore can access all Camunda Platform 8 clusters of this organization.
+Client credentials are created for an organization, and therefore can access all Camunda Platform 8 clusters of this organization.
 
-A client can have one or multiple of the following permissions:
+Scopes define the access for client credentials. A client can have one or multiple of the following permissions:
 
+- **Get clients**: Retrieve all Zeebe clients of the organization.
```

Comment:
That's fine for me. In Console we still show `Zeebe Client` but since it's the only client reference on this modal it should be clear enough.